tests/foreman/cli/test_oscap.py

```diff
@@ -0,0 +1,710 @@
+        3. Oscap-cli hammer plugin installed.
+
+        @steps:
+
+        1. Login to hammer shell as admin.
+        2. Execute the "scap-content" command with info as sub-command.
+        3. Pass valid "ID" of scap-content as argument.
+
+        @Assert: The info of the scap-content is listed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_scap_content_info_viewer_role(self):
+        """View info of scap content with viewer role
+
+        @id: 15eb035b-d301-4dbd-b66a-c4621d2003a3
+
+        @setup:
+
+        1. Oscap should be enabled.
+        2. Default content should already be populated.
+        3. Oscap-cli hammer plugin installed.
+
+        @steps:
+
+        1. Login to hammer shell with user with viewer role.
+        2. Execute the "scap-content" command with info as sub-command.
+        3. Pass valid parameters.
+
+        @Assert: The info of the scap-content is not listed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_scap_content_info(self):
+        """View info of scap content with invalid ID as parameter
+
+        @id: 86f44fb1-2e2b-4004-83c1-4a62162ebea9
+
+        @setup:
+
+        1. Oscap should be enabled.
+        2. Default content should already be populated.
+        3. Oscap-cli hammer plugin installed.
+
+        @steps:
+
+        1. Login to hammer shell as admin.
+        2. Execute the "scap-content" command with info as sub-command.
+        3. Pass invalid "ID" of scap-content as argument.
+
+        @Assert: The info of the scap-content is not listed.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_scap_content_with_valid_title(self):
+        """Create scap-content with valid title
+
+        @id: 68e9fbe2-e3c3-48e7-a774-f1260a3b7f4f
+
+        @setup:
+
+        1. Oscap should be enabled.
+        2. Oscap-cli hammer plugin installed.
+        3. Scap data stream ".xml" file.
+
+        @steps:
+
+        1. Login to hammer shell.
+        2. Execute "scap-content" command with "create" as sub-command.
+        3. Pass valid parameters.
+
+        @Assert: The scap-content is created successfully.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_negative_create_scap_content_with_invalid_title(self):
+        """Create scap-content with invalid title
+
+        @id: 90a2590e-a6ff-41f1-9e0a-67d4b16435c0
+
+        @setup:
+
+        1. Oscap should be enabled.
+        2. Oscap-cli hammer plugin installed.
+        3. Scap data stream ".xml" file.
+
+        @steps:
+
+        1. Login to hammer shell.
+        2. Execute "scap-content" command with "create" as sub-command.
+        3. Pass valid parameters and invalid title.
+
+        @Assert: The scap-content is not created.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_scap_content_with_valid_originalfile_name(self):
```

Comment:
I didn't get `originalfile` name ? Can we give any name ? or is it something specific. ? Lets talk on this once.